bio-primer-removal/main.py

```diff
@@ -0,0 +1,95 @@
+from pathlib import Path
+from zipfile import ZipFile, ZIP_DEFLATED
+
+import logging
+
+from coretex import Experiment, CustomDataset, CustomSample, folder_manager
+from coretex.project import initializeProject
+from coretex.bioinformatics import cutadaptTrim
+
+
+def loadSingleEnd(dataset: CustomDataset) -> list[Path]:
```

Comment:
let's use `SequenceDataset` instead of `CustomDataset` here. `SequenceSample` has `sequencePath` property inside which returns path to any of the following files: .fasta, .fastq, .fa, .fq